### ISSUE TITLE
Sharing multiple files to other apps

### ISSUE THREAD

**Issue by ncyclon** [nextcloud/android#677]
Is it possible to realize sharing multiple selected files to others application. Few photo to hangout, for example. 

There is possible to do it only for one selected file.

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky do you know if this can (easily) implemented?

**Comment by mario** [comment]
Yes it can. I believe there was even a PR hanging somewhere, but anyway ... it's easy, just takes time :P

**Comment by tobiasKaminsky** [comment]
The only PR I have found is https://github.com/nextcloud/android/pull/478 but this is for sending resized images.
If I have not overlooked something this is has to be freshly implemented.

**Comment by daniel-jirca** [comment]
This should be part of a pretty natural experience for a file management app/server, especially considering that many people use nextcloud for storing pictures and would want to share those directly through other apps.
The single file sharing functionality is already implemented, is it possible to prioritize this issue considering the fact that it wouldn't require a big effort and the issue is already 3 years old?

**Comment by joshtrichards** [comment]
@johnnyasantoss suggested in #7977:

> Select multiple files and share them with any other app. It shouldn't matter whether the file is downloaded or not. In the case that the file is missing it should show a progress bar of some kind in a modal representing the download of the missing files out of the selected ones. After downloading or if the files were already synced then it should open the default android sharing activity.
